params/mavproxy.param.default

```diff
@@ -2,10 +2,10 @@
 --load-module='GPSInput,DepthOutput'
 --source-system=200
 --cmd="set heartbeat 0"
+--cmd="param set SERIAL0_PROTOCOL 2"
```

Comment:
We should also add `--mav20`

Reply:
It's already there - from previous update